pkgs/applications/emulators/mgba/default.nix

```diff
@@ -36,6 +37,7 @@ stdenv.mkDerivation (finalAttrs: {
   nativeBuildInputs = [
     cmake
     pkg-config
+    wrapGAppsHook
```

Comment:
Double wrapping too dangerous.

Reply:
I guess I'm supposed to do something like https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/applications/misc/keepassxc/default.nix#L93-L95?